src/systems/frequency_system.C

```diff
@@ -112,14 +112,18 @@ void FrequencySystem::init_data ()
   // make sure we have frequencies to solve for
   if (!_finished_set_frequencies)
     {
+      // not supported for now
+      if (parameters.have_parameter<unsigned int> ("n_frequencies"))
+        libmesh_error_msg("ERROR: Not supported");
```

Comment:
Why not?  Seems like a straightforward behavior even if we lack the proper test coverage for it.

Reply:
there's quite a few places to make changes to make it work

Reply:
like in `set_frequencies_by_range` do we set the es.parameters now, or switch to the local system, or both?

Reply:
Okay, that's a good reason.  Setting just local system parameters breaks backwards compatibility, setting just es means we might not actually change the parameters used, and setting both means we lose the new flexibility.  I'd say change this to `libmesh_not_implemented_msg()` but there's no need to actually change the behavior now.